src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/openapi/KeysApiController.java

```diff
@@ -225,14 +226,14 @@ private Key getKeyFromService(String keyId) {
     @Override
     @PreAuthorize("hasAnyAuthority('DELETE_KEY', 'DELETE_AUTH_KEY', 'DELETE_SIGN_KEY')")
     @AuditEventMethod(event = DELETE_KEY)
-    public ResponseEntity<Void> deleteKey(String keyId) {
+    public ResponseEntity<Void> deleteKey(String keyId, Boolean ignoreWarnings) {
```

Comment:
Would be good to add javadocs to old public void `deleteKey(String keyId) {` so that it explains the ignoreWarnings = true deal. (weird, I thought checkstyle requires javadocs for public methods. It is good practice to have those anyhow)

Maybe also consider changing old method name to something like deleteKeyAndIgnoreWarnings, or removing it altogether and using deleteKey(keyId, true) instead. Does having a shorthand version, without boolean parameter, make things better or worse? Maybe without it code might be more clear and more explicit about what is the intended action. Well, not sure about that one really.

Reply:
Changed the old method name to `deleteKeyAndIgnoreWarnings` and updated javadocs.